### PR TITLE
RNA STAR: Expose genomeSAindexNbases parameter

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This PR exposes the genomeSAindexNbases parameter.

### CODE REVIEW

File: tools/rgrnastar/macros.xml [hunk 1]
```diff
@@ -163,10 +163,8 @@
     ]]></token>
     <xml name="ref_selection">
         <param argument="--genomeFastaFiles" type="data" format="fasta" label="Select a reference genome" />
-        <!-- Currently, this parameter is not exposed in the wrapper,
-             but used only in the tests to avoid excessive index sizes for
-             the tiny test genomes. -->
-        <param name="genomeSAindexNbases" type="hidden" value="" />
+     
+        <param argument="--genomeSAindexNbases" type="integer" min="2" max="16" value="14" label="Length of the SA pre-indexing string" help="Typically between 10 and 15. Longer strings will use much more memory, but allow faster searches. For small genomes, the parameter --genomeSAindexNbases must be scaled down to min(14, log2(GenomeLength)/2 - 1)"/>
```
Comment (bernt-matthias) [inline tools/rgrnastar/macros.xml:167]: I think you also extend the command section to actually use the parameter. Also some kind of test for the new parameter would be nice.

**Review by bernt-matthias — CHANGES_REQUESTED** [review]

File: tools/rgrnastar/macros.xml (hunk 1 above)
Comment (gallardoalba) [reply tools/rgrnastar/macros.xml:167]: Sure, I'll include it.

**Comment by wm75** [comment]
There is the related #3496. Maybe @pavanvidem and you @gallardoalba can coordinate which change should go in first, or whether you want to merge your work into one PR?

**Comment by gallardoalba** [comment]
> There is the related #3496. Maybe @pavanvidem and you @gallardoalba can coordinate which change should go in first, or whether you want to merge your work into one PR?

I can add my modification to Pavan's branch :)

> There is the related #3496. Maybe @pavanvidem and you @gallardoalba can coordinate which change should go in first, or whether you want to merge your work into one PR?

I'll open a PR against the @pavanvidem branch.

**Pull request closed without merging**